app/views/special_event/HoC2018InterstitialModal.coco.coffee

```diff
@@ -11,7 +12,7 @@ module.exports = class HoC2018InterstitialModal extends ModalComponent
   initialize: ->
     @propsData = {
       clickStudent: () => @hide(),
-      clickTeacher: () => application.router.navigate("/teachers/hour-of-code?activity=#{@options.activity or 'ai-league'}", { trigger: true }),
+      clickTeacher: () => application.router.navigate("/teachers/hour-of-code" + utils.isCodeCombat ? "?activity=#{@options.activity or 'ai-league'}" : "", { trigger: true }),
```

Comment:
@adamkecskes this one turns out to be buggy because CoffeeScript uses `if COND then A else B` for a ternary operator instead of `COND ? A : B` and ends up navigating us to `/teachers/hour-of-codetrue`. I fixed it [here](https://github.com/codecombat/codecombat/commit/5c37218ff445bc62c1843f939ef7c686debcdf39#diff-5fd8d15656e0a6be401b4dd2ee2bf2a14e13e17feadd99eeac0b3dc4f79721a4R15), but just a heads up to differentiate when merging them CoffeeScripts.